cmd/helm/deployment.go

```diff
@@ -90,3 +98,18 @@ func list(c *cli.Context) error {
 	format.List(list)
 	return nil
 }
+
+func deleteDeployment(c *cli.Context) error {
+	args := c.Args()
+	if len(args) < 1 {
+		return errors.New("First argument, deployment name, is required. Try 'helm get --help'")
+	}
+	for _, name := range args {
+		deployment, err := NewClient(c).DeleteDeployment(name)
+		if err != nil {
+			return err
+		}
+		format.Info("Deleted %q at %s", name, deployment.DeletedAt)
```

Comment:
That is a good question. Right now, it's always wrong. I was thinking the timestamp would make a decent piece of output (since it seems that these may take several seconds to delete). But I'm not devoted to this.